skyportal/handlers/api/photometry.py

```diff
@@ -370,6 +370,11 @@ def _compile_photometry_values(element, compiler, asfrom=False, **kw):
 
             def coerced_value(elem, column):
                 literal_value = compiler.render_literal_value(elem, column.type)
+                if literal_value == 'inf':
+                    literal_value = '+infinity'
```

Comment:
I think this needs to be string-escaped 
```suggestion
                    literal_value = "'infinity'"
```